RELEASENOTES.md

```diff
@@ -0,0 +1,3 @@
+### New in 5.0.0
```

Comment:
Because we still need to push the git tag corresponding to the actual release. So it's not useful if Appveyor does the release and doesn't push the git tag. So if I instead build it locally, I can both manage the release notes and the tagging via the build script.